test/forms/TransportSpec.scala

```diff
@@ -84,10 +84,15 @@ class TransportSpec extends BaseSpec {
 
       "values are correct" in {
 
-        val inputData = Transport(Sea, "PL")
-        val errors = Transport.form().fillAndValidate(inputData).errors
+        val inputData1 = Transport(Sea, "PL")
+        val errors1 = Transport.form().fillAndValidate(inputData1).errors
+
+        errors1.length must be(0)
+
+        val inputData2 = Transport(Sea, "XK")
+        val errors2 = Transport.form().fillAndValidate(inputData2).errors
```

Comment:
```suggestion
        Transport.form().fillAndValidate(inputData1).errors should be (empty) 
```

Reply:
@benjaminparker I have addressed all the above comments. Please review.